code-challenges.test.js

```diff
@@ -9,28 +9,120 @@
 // Run the file with the following command: $ yarn jest
 
 // --------------------1) Prompt: Create a function that takes in a number (greater than 2) and returns an array containing the Fibonacci sequence. The length of the array is determined by the argument of the function.
+function genFib(length) {
+    if (typeof length !== 'number' || length <= 2) {
+      throw new Error('Input must be a number greater than 2')
+    }
+  
+    const fibArray = [0, 1]
+  
+    for (let i = 2; i < length; i++) {
+      const nextFib = fibArray[i - 1] + fibArray[i - 2]
+      fibArray.push(nextFib)
+    }
+  
+    return fibArray
+}
```

Comment:
Good work taking care of edge cases. Consider looking at line 3 condition. The fibonacci needs at least 2 numbers to have a value in index  0 and 1.